opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -151,63 +151,63 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
             SpanContext extractedContext = tracer.extract(Format.Builtin.HTTP_HEADERS,
                     new HttpServletRequestExtractAdapter(httpRequest));
 
-            final ActiveSpan span = tracer.buildSpan(httpRequest.getMethod())
+            final Scope scope = tracer.buildSpan(httpRequest.getMethod())
                     .asChildOf(extractedContext)
                     .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_SERVER)
-                    .startActive();
+                    .startActive(false);
 
-            httpRequest.setAttribute(SERVER_SPAN_CONTEXT, span.context());
+            httpRequest.setAttribute(SERVER_SPAN_CONTEXT, scope.span().context());
 
             for (ServletFilterSpanDecorator spanDecorator: spanDecorators) {
-                spanDecorator.onRequest(httpRequest, span);
+                spanDecorator.onRequest(httpRequest, scope.span());
             }
 
             try {
                 chain.doFilter(servletRequest, servletResponse);
                 if (!httpRequest.isAsyncStarted()) {
                     for (ServletFilterSpanDecorator spanDecorator : spanDecorators) {
-                        spanDecorator.onResponse(httpRequest, httpResponse, span);
+                        spanDecorator.onResponse(httpRequest, httpResponse, scope.span());
                     }
                 }
                 // catch all exceptions (e.g. RuntimeException, ServletException...)
             } catch (Throwable ex) {
                 for (ServletFilterSpanDecorator spanDecorator : spanDecorators) {
-                    spanDecorator.onError(httpRequest, httpResponse, ex, span);
+                    spanDecorator.onError(httpRequest, httpResponse, ex, scope.span());
                 }
                 throw ex;
             } finally {
                 if (httpRequest.isAsyncStarted()) {
-                    final ActiveSpan.Continuation cont = span.capture();
+                    final Span cont = scope.span();
```

Comment:
Do we need to define it here? What about passing the span directly to ` tracer.scopeManager().activate(cont)`

Reply:
The scope could be closed - which makes accessing the span potentially unpredicatable - I don't know if it is defined whether the span remains accessible after the scope is closed?

Reply:
> the scope could be closed 

Could you please explain it more? This scope is accessible only from this class.

https://github.com/pavolloffay/opentracing-java/blob/4ae4656919181255d70fc67e9ddacc12d94a77a2/opentracing-api/src/main/java/io/opentracing/Scope.java#L40

Reply:
I guess there is nothing in the method that says Span is not accessible after close so will change.